apps/talk.sh

```diff
@@ -486,3 +488,33 @@ else
     msg_box "Congratulations, everything is working as intended! The installation succeeded.\n\nLogging can be found by typing: journalctl -lfu signaling"
     exit 0
 fi
+
+####### Talk recording
+print_text_in_color "$ICyan" "Setting up Talk recording..."
```

Comment:
I would recommend to make this optional as talk recording is very resource intensive.

Reply:
Yeah sure, been thinking about that too.